src/locales/data.ts

```diff
@@ -520,7 +520,7 @@ const data = {
       America: "America",
     },
     mmrLeagueRanges: {
-      MMR_0: "all",
+      MMR_0: "All",
```

Comment:
These things are handled in the google sheet, and will get overwritten the next time the script is run.

Reply:
Yep, I updated it in the sheet already, just did it locally to test/confirm visually.